tests/acceptance/test_pre_detection.py

```diff
@@ -27,84 +44,126 @@
 ]
 
 
+def test_events_pre_detected_runs_without_error(tmp_path: Path):
+    config = get_default_logprep_config(pipeline, with_hmac=False)
+    input_file_path = tmp_path / "input.json"
+    config.input["jsonl"]["documents_path"] = str(input_file_path)
+    config_path = tmp_path / "generated_config.yml"
+    config_path.write_text(config.as_yaml())
+    proc = start_logprep(config_path)
+    output = proc.stdout.readline().decode("utf8")
+
+    while True:
+        assert not re.search("Invalid", output)
+        assert not re.search("Exception", output)
+        assert not re.search("critical", output)
+        assert not re.search("Error", output)
+        assert not re.search("ERROR", output)
+        if re.search("Startup complete", output):
+            break
+        output = proc.stdout.readline().decode("utf8")
+    stop_logprep(proc)
+
+
+def test_events_pre_detected_correctly(tmp_path: Path):
+    config = get_default_logprep_config(pipeline, with_hmac=False)
+    config_path = tmp_path / "generated_config.yml"
+    config_path.write_text(config.as_yaml())
+
+    output_path = Path(config.output["jsonl"]["output_file"])
+    input_file_path = Path(config.input["jsonl"]["documents_path"])
+
+    proc = start_logprep(config_path)
+    wait_for_output(proc, "no documents left")
+    stop_logprep(proc)
+
+    assert output_path.read_text("utf8"), "output is not empty"
+
+    with output_path.open("r", encoding="utf8") as f:
+        output_data = [json.loads(line) for line in f]
+    with input_file_path.open("r", encoding="utf8") as f:
+        input_data = [json.loads(line) for line in f]
```

Comment:
```suggestion
    output_data = map(json.loads, output_path.read_text("utf8").splitlines())
    input_data = map(json.loads, input_file_path.read_text("utf8").splitlines())
```

Consider using this, because it is more readable and pythonic